docs/common/custom.md

```diff
@@ -34,7 +34,7 @@ For example, if I had an NFT custom contract and I wanted to call the `balance`
 
 ```python
 # Any arguments passed to the balance function go into the args of "call"
-custom.functions.balance(1).call()
+contract.functions.balance(1).call()
```

Comment:
Need to update this example and the one below right?

Reply:
Ah yes was editing the doc but hadn't pushed up 😄